monitoring/enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="enabling-alert-routing-for-user-defined-projects"]
+= Enabling alert routing for user-defined projects
+include::modules/common-attributes.adoc[]
+:context: enabling-alert-routing-for-user-defined-projects
+
+toc::[]
+
+:FeatureName: Alert routing for user-defined projects 
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+In {product-title} {product-version}, a cluster administrator can enable alert routing for user-defined projects. 
```

Comment:
I like this explanation of the relationship between the administrator and the user. My only concern: if I'm not an administrator, I might read this and think this content, and the following procedures, are not for me; they're for admins. 

What do you think about keeping this intro paragraph the same, but editing the next sentence (before the bullet list). Something like:

`When you alert routing for user-defined projects, note the following features:`

I think a rephrase would signal to me, "okay, cool, I should be reading this content, even though I'm not a cluster administrator. These features and procedures are for me." 

What do you think?

Reply:
I think this phrasing would still speak to admins as well. Basically, the use of second person includes all users, regardless of their role.

Reply:
If all these procedures are only intended for a cluster administrator, and not really for other roles, you can rephrase to:

`As a cluster administrator in OpenShift Container Platform Branch Build, you can enable alert routing for user-defined projects. After you enable alert routing, your users can configure alert routing for their user-defined projects....`